reach/reach.py

```diff
@@ -628,6 +628,48 @@ def _most_similar_batch(
                 word_index = np.flip(np.argsort(sims_for_word))
                 yield [(self.indices[indices[idx]], sims_for_word[idx]) for idx in word_index]
 
+    def indices_threshold(
```

Comment:
Maybe rename this to read: "nearest_neighbor_indices_threshold"? Within reach, we distinguish between `most_similar`, which is where you pass item names, and `nearest_neighbor`, which is where you pass vectors.

Reply:
Additionally, the original `threshold` also has a `most_similar` counterpart. I'm not sure if it's that interesting here, just something to note.